torch/csrc/autograd/FunctionsManual.cpp

```diff
@@ -3237,12 +3237,11 @@ Tensor as_strided_scatter_backward(
   // take the perf hit and contiguify grad for now.
   auto grad_ = grad.contiguous();
   auto grad_slice = grad_.as_strided_symint(sizes, strides, storage_offset);
-  auto result =
-      grad_.new_zeros_symint(input_geometry.sym_sizes())
-          .as_strided_symint(
-              input_geometry.sym_sizes(), input_geometry.sym_strides());
-  auto result_slice =
-      result.as_strided_symint(sizes, strides, std::move(storage_offset));
+  auto result_buffer = grad_.new_zeros_symint(input_geometry.sym_sizes());
```

Comment:
maybe cc @albanD - it looks like we were in fact using "non-compositional as_strided + mutation" in the backward formula for `as_strided_scatter` (lol), which failed a few tests.

I think this change is a no-op, but tagging you just in case.

Reply:
Pretty cool code :)